src/app.rs

```diff
@@ -333,6 +333,16 @@ pub fn build_app() -> App<'static, 'static> {
                        '{/.}': basename without file extension",
                 ),
         )
+        .arg(
+            Arg::with_name("has-match")
+                .long("has-match")
+                .conflicts_with_all(&["exec", "exec-batch", "list-details"])
```

Comment:
I believe there might be a few more options it conflicts with:
- `--max-results` doesn't really make sense in combination with `has-match`, right?
- `--prune`: not sure.
- Options like `--print0`, `--absolute-path`, … that affect the output of `fd` don't make a lot of sense in combination with `--has-match` either, but it's probably best to *not* mark them as conflicting.

Reply:
Ah yeah, I'll add `--max-results` as conflicting too.

For `--prune`, I think it's similar to `--print0` and `--absolute-path` in the sense that including it won't affect the output of `--has-match`, so there's no need to mark it as conflicting in my eyes.